lib/kredis/attributes.rb

```diff
@@ -38,16 +38,16 @@ def kredis_json(name, key: nil, config: :shared)
       kredis_connection_with __method__, name, key, config: config
     end
 
-    def kredis_list(name, key: nil, typed: :string, config: :shared)
-      kredis_connection_with __method__, name, key, typed: typed, config: config
+    def kredis_list(name, key: nil, typed: :string, config: :shared, changed: ->(list){})
```

Comment:
Should just use nil as the default, so we're not creating new enclosures when they're not used.